docs/en/operation/Journal.md

```diff
@@ -8,6 +8,19 @@ priority: 0
 
 TODO(andrew): Add more information
 - Talk about log files and checkpoints, how to manage the size
+- Lead with what the journal is, and its fault tolerance guarantees
+- Introduction should describe purpose of doc, which is administering the journal of an Alluxio cluster.
+
+Outline:
+# Configuration
+# Formatting the Journal
+# Operations
+## Manually Backing up the Journal
+## Restoring from a Backup
+## Changing Masters
+# Advanced
+## Running Alluxio in High Availability (HA) Mode
+## Space requirements (log files, checkpoints)
```

Comment:
can we add common issues in a troubleshooting section to include 
- how to handle incomplete journal files (due to HDFS issues when writing journals)
- what if we see too many log files generated? (write-delete intensive workloads)
- how to debug readable journal entries?

Reply:
- Since 1.7, we automatically recover hdfs leases, so users don't need to worry any more about incomplete journal files.
- This will be covered by the "Space requirements" section, which will talk about how much space is needed, and how to keep it from growing too large. I've updated the outline to include this.
- I don't think admins or users should need to read the journal in plain text. Issues that require reading the journal can only be addressed by fixing code-level bugs. This should probably go to the "Alluxio Contributor Resources" section instead for people who want to develop on the journal

Reply:
SG